qt/python/mantidqt/project/project.py

```diff
@@ -167,6 +185,16 @@ def _offer_save_message_box(self, parent):
         else:
             return QMessageBox.No
 
+    @staticmethod
+    def offer_large_size_confirmation():
```

Comment:
This probably needs to be a private method
```suggestion
    def _offer_large_size_confirmation():
```